# A timestamped 2d point with time

Header header
float64 x
float64 y
time t
